### DISCUSSION TITLE
Add search tab nodes function

### DISCUSSION THREAD

**Discussion by zeroarst** [Able-Bear-Software/tab-nodes-tree#75] [Ideas / Feature Request]
Although Chrome has a built-in search function (Ctrl+Shift+A), it’s still worthwhile to build a native version with more flexible settings for users.

Proposal:
1. Add a setting to toggle "Always show search input field."
2. Allow the search input field to be shown/hidden via a keyboard shortcut.
3. Add a setting to toggle the position of the search input field (top or bottom).
4. Highlight keywords in search results, with an optional highlight style setting.
5. Pro version only.

Further thoughts are welcome!

**Comment by reysu47** [comment]
+1, would like to have search functionality, this is really useful when you have a bunch of tabs.

**Comment by Tab-Nodes-Tree** [comment]
I will try see if I can implement and include it with v1.7.0.
#136
